### PR TITLE
Only run on hosts with the escluster label

### PR DESCRIPTION
**Pull request by adamgraves-choices**

### CODE REVIEW

File: rancher-config/docker-compose.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ services:
     es-master:
         labels:
             io.rancher.scheduler.global: true
+            io.rancher.scheduler.affinity:host_label: escluster
```
Comment (st3v3nhunt) [inline rancher-config/docker-compose.yml]: Reading the [docs](http://rancher.com/docs/rancher/latest/en/cattle/scheduling/#values) it looks like the value needs to be in the format `key=value`.

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
A question about the value and whether it should be key=value.

File: rancher-config/docker-compose.yml (hunk 1 above)
Comment (adamgraves-choices) [reply rancher-config/docker-compose.yml]: I don't think that it is needed. I'll check in the dev environment and see what happens.

Comment (adamgraves-choices) [reply rancher-config/docker-compose.yml]: You're right. the label does need to be `key=value`

**Review by st3v3nhunt — APPROVED** [review]
LGTM

**Pull request merged**